sections/06_discussion.md

```diff
@@ -6,9 +6,28 @@ can be further pruned.*
+
+#### Formulation of classification labels
+Genome-wide continuous signals are commonly formulated into classification labels through signal peak detection. For example, peaks in ChIP-seq signal are used to identify locations of TF binding and histone modifications. Such procedures rely on thresholding criteria to define what constitutes a peak in the signal. This inevitably results in a set of signal peaks that are close to the threshold, not sufficient to constitute a positive label, and too similar to positively labeled examples to constitute a negative label. To avoid an arbitrary label for these example they may be labeled as “ambiguous”. Ambiguously labeled examples can then be ignored during model training and evaluation of recall and FDR. The correlation between model predictions on these examples and their signal values can be used to evaluate if the model correctly ranks these examples between positive and negative examples.
+
+#### Formulation of a performance upper bound
+Genome-wide signals across experiments can lead to different sets of positive examples. When experimental replicates do not completely agree, a 100% recall at 25% FDR is not possible. The upper bound on the recall is the fraction of positive examples that are in agreement across experiments. This fraction will vary depending on the available experimental data. For example, reproducibility for experimental replicates from the same lab is typically higher than experimental replicates across multiple labs. One way to handle the range of reproducibility is the use of multiple reproducibility criteria such as reproducibility across technical replicates, biological replicates from the same lab, and biological replicates from multiple labs.
+
+
+*Is progress in some biomedical areas slowed when new predictions (e.g. from
```

Comment:
Upon further consideration, I agree with you about dropping the italics here.  I have nothing to add beyond the obvious.